deployment/overlays/tls-backend/kustomization.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base/maas-api
+  - networking/token-service-destinationrule.yaml
+  - authorino-deployment.yaml
+  - authorino-volume.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+  - name: maas-api-ca-cert
+    namespace: kuadrant-system
+    files:
+      - ca-bundle.crt=certs/maas-api-server.crt
```

Comment:
The kustomization.yaml references `certs/maas-api-server.crt` which doesn't exist in CI environment. Certificate files are correctly excluded by `.gitignore`, but kustomize validation requires files to exist. Thus, the CI fails.